policy-controller/k8s/status/src/index.rs

```diff
@@ -51,7 +49,7 @@ pub struct Controller {
     updates: mpsc::Receiver<Update>,
     patch_timeout: Duration,
 
-    /// True if this policy controller is the leader — false otherwise.
```

Comment:
Lets avoid grammar changes unless they improve clarity.

Reply:
Ah, yeah, I've got my IDE set to be almost irritatingly pedantic[¹](#um-actually) and fixing the squigglies is essentially force of habit at this point. I'll revert the change.

<details id="um-actually">
  <summary>visual reference</summary>
  ![my ide](https://c.tenor.com/btLFJppmtiIAAAAC/tenor.gif)
</details>